ui/page/governance/agenda_vote_modal.go

```diff
@@ -0,0 +1,307 @@
+package governance
+
+import (
+	"sort"
+
+	"gioui.org/font/gofont"
+	"gioui.org/layout"
+	"gioui.org/text"
+	"gioui.org/widget"
+	"gioui.org/widget/material"
+
+	"github.com/planetdecred/dcrlibwallet"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/page/components"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+type agendaVoteModal struct {
+	*load.Load
+	modal decredmaterial.Modal
+
+	LiveTickets []*dcrlibwallet.Transaction
```

Comment:
Not used outside this package. Also, these tickets aren't exactly live, let's call them votable, and add a comment explaining what that means.
```suggestion
	votableTickets []*dcrlibwallet.Transaction // tickets that have not been spent by a vote or revocation (unspent) and that have not expired (unexpired).
```